get-new-tool-versions/verify-new-tool-version-added-to-image.ps1

```diff
@@ -0,0 +1,43 @@
+<#
+.SYNOPSIS
+Check and return list of new available tool versions
+
+.PARAMETER ToolName
+Required parameter. The name of tool for which parser is available (Node, Go, Python, Xamarin)
+#>
+
+param (
+    [Parameter(Mandatory)] [string] $ToolName
+)
+
+if ($ToolName -eq "Python") {
+    $builtStableMinorVersions = ((Invoke-RestMethod "https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json") |
+        Where-Object {$_.stable -eq $true}).version |
```

Comment:
1. I think we can remove ()  and move .version to Foreach-Object
2. nitpick: Where-Object -> `Where-Object State -eq $true`

Reply:
1.1

Reply:
1.2 Ohh, it works actually. Will update